romana-install/roles/romana/install-agent/tasks/romana_from_github.yml

```diff
@@ -2,10 +2,33 @@
   command: rsync -az "{{ romana_master_ip }}:/var/tmp/gopath/bin/{{ item }}" "/var/tmp/{{ item }}"
   with_items:
     - agent
+    - romana
+
+- name: Copy policy-agent from master
+  command: rsync -az "{{ romana_master_ip }}:/var/tmp/gopath/src/github.com/romana/core/pkg/util/policy/agent/agent.py" "/var/tmp/agent.py"
 
 - name: Install files
   become: true
   become_user: root
   command: install -o root -g root -m 0755 "/var/tmp/{{ item }}" "{{ romana_bin_dir }}/{{ item }}"
   with_items:
     - agent
+    - romana
+    - agent.py
+
+- name: Install Agent Proxy Add Policy
+  become: true
+  become_user: root
+  template: src="config/agent_proxy_add_policy.sh" dest="{{ romana_bin_dir }}/" mode=0777
```

Comment:
Right. I suppose `/etc/romana/hooks/agent/` would be a good spot until we think of something better. (The root config would also need to be updated)